build.gradle

```diff
@@ -40,7 +40,7 @@ targetCompatibility = 1.7
 ext {
   generatedSrcDir = "${projectDir}/generated/src"
 
-  grpcVersion = '1.0.3'
+  grpcVersion = '1.2.0'
```

Comment:
I want to do a unified bump to grpc 1.2 (see https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1691), which involves updating the grpc-generated packages to grpc 1.2 also. @shinfan has just finished moving the grpc package generation to toolkit, so we can do this update now. I'm thinking we might want to pull out the changes in this PR necessary for 1.2, and do the maxInboundMessageSize after. @shinfan is currently assigned to do the upgrade - if someone else does it first, please coordinate with him.